index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
PLease change the name of this index.hml to something more specific or else the merge will delete everything else under index.html